srcpkgs/spotify-player/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'spotify-player'
+pkgname=spotify-player
+version=0.13.1
+revision=1
+archs="*"
```

Comment:
this is the default value, remove it

Reply:
please remove the whole line and fix cross compilation by putting -devel packages to make depends instead of hostmakedepends

Reply:
The -devel packages are aleady in the make depends? And remove the arch line entirely?
Thank you for all the help, I will know all of this for next time :smile:

Reply:
oh, so the other way around, move pkg-config to hostmakedepends, it's not a library and provides a binary that has to run on the host